llvm/test/CodeGen/X86/GlobalISel/legalize-icmp-vec.mir

```diff
@@ -0,0 +1,24 @@
+# RUN: llc -mtriple=x86_64-linux-gnu -run-pass=legalizer -global-isel-abort=2 -pass-remarks-missed='gisel*' %s -o - 2>%t | FileCheck %s
+# RUN: FileCheck -check-prefix=ILLEGAL %s < %t
+
+# ILLEGAL: remark: <unknown>:0:0: unable to legalize instruction: %2:_(<4 x s1>) = G_ICMP intpred(sle), %0:_(<4 x s64>), %1:_ (in function: test_icmp_v4i64)
```

Comment:
Is this patch only to fix the first crash reported in #86203?

Reply:
Please can you add a reference to PR86203?

Reply:
@phoebewang it's not a crash, actually. The crash happens if we use only `-global-isel`. This option forces llc to use only GlobalISel. The correct way is to use `-fglobal-isel` with clang or `-global-isel -global-isel-abort=2` with llc. It allows to fallback to SelectionDAG if something is unsupported in GlobalISel. Without this fix we hit the assert and are not able to fallback to SelectionDAG.

@RKSimon done.